libflux/src/flux/semantic/bootstrap.rs

```diff
@@ -0,0 +1,288 @@
+use std::collections::HashMap;
+use std::fs;
+use std::io;
+use std::path::PathBuf;
+
+use crate::ast;
+use crate::parser;
+use crate::semantic::analyze::analyze_file;
+use crate::semantic::env::Environment;
+use crate::semantic::fresh::Fresher;
+use crate::semantic::import::Importer;
+use crate::semantic::infer;
+use crate::semantic::infer::Constraints;
+use crate::semantic::nodes;
+use crate::semantic::nodes::infer_file;
+use crate::semantic::sub::Substitutable;
+use crate::semantic::types;
+use crate::semantic::types::{MonoType, PolyType, Property, Row};
+
+use walkdir::WalkDir;
+
+const PRELUDE: [&str; 2] = ["universe", "influxdata/influxdb"];
+
+type Error = String;
+
+impl From<nodes::Error> for Error {
+    fn from(err: nodes::Error) -> Error {
+        err.to_string()
+    }
+}
+
+impl From<types::Error> for Error {
+    fn from(err: types::Error) -> Error {
+        err.to_string()
+    }
+}
+
+#[allow(dead_code)]
+// Recursively parse all flux files within a directory.
+fn parse_flux_files(path: &str) -> io::Result<Vec<ast::File>> {
```

Comment:
Should these methods live in the parser module?

Reply:
For right now, let's keep it as a utility method for typing the standard lib. And we can update it for user defined packages.